notebooks/290-magika-content-type-recognition/290-magika-content-type-recognition.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #150.            self. _get_results_from_features = partial(self._get_results_from_features_topk, top_k=top_k)
</pre><p><code>top_k </code>is not provided in <code>identify_path_topk</code> and <code>identify_paths_topk </code>methods. </p><p> </p><p>I believe it would be better to replace all these 3 methods by one, where a type of provided source will be checked and a corresponding internal method will be called.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>